BarrysPeanutsAkkaJava/src/main/java/barryspeanuts/App.java

```diff
@@ -0,0 +1,104 @@
+package barryspeanuts;
+
+import akka.actor.ActorRef;
+import akka.actor.ActorSystem;
+import akka.actor.Props;
+import barryspeanuts.actor.ShoppingCartActor;
+import barryspeanuts.model.Address;
+import barryspeanuts.model.CreditCard;
+import barryspeanuts.model.Customer;
+import barryspeanuts.model.PurchaseItem;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class App {
+  public static void main(String[] args) {
+    Logger logger = LoggerFactory.getLogger(ShoppingCartActor.class);
+
+    logger.info("Starting Barry's Gourmet Peanuts.");
+
+    akka.actor.ActorSystem system = ActorSystem.create("BarrysPeanutsSystem");
+
+    Address address = new Address("123 Main Street", "Apt 1", "Anytown", "CA", "99999-9999", "USA");
+
+    Customer customer =
+        new Customer(
+            UUID.randomUUID(), "Barney", "Rubble", "barney@rubble.com", "310 878 9999", address);
+
+    // Create some purchase items to add to the shopping cart
+    List<PurchaseItem> purchaseItems = new ArrayList<>();
+
+    PurchaseItem purchaseItem1 =
```

Comment:
Does PurchaseItem really need to know about the customer and addresses?

Reply:
I was trying to avoid the labor of implementing a customerID and then doing all that management.